gui/wxpython/modules/import_export.py

```diff
@@ -122,16 +125,17 @@ def __init__(
         # buttons
         #
         # cancel
+        self._DEFAULT_CLOSE_BTN_TEXT = "Close dialog"
```

Comment:
This should be translatable, here you are calling SetToolTip with the translated version and later with untranslated version.